packages/palette/src/elements/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { FC } from "react"
+import { Box, Flex } from "@artsy/palette"
+import styled, { css } from "styled-components"
+import { zIndex } from "styled-system"
+import { FocusOn } from "react-focus-on"
+
+export interface DrawerProps {
+  open: boolean
+  anchor?: "left" | "right"
+  zIndex?: number
+  onClose?(): void
+}
+
+export const Drawer: FC<DrawerProps> = ({
+  children,
+  anchor = "right",
+  zIndex = DEFAULT_DRAWER_Z_INDEX,
+  open,
+  onClose,
+}) => {
+  const openClass = open ? "open" : ""
+
+  return (
+    <Container zIndex={zIndex} anchor={anchor} className={openClass}>
+      <Focus onClickOutside={onClose} enabled={open} onEscapeKey={onClose}>
+        <Content
+          backgroundColor="white100"
+          height="100%"
+          width={["100%", "auto"]}
+          overflowX="hidden"
+          overflowY="scroll"
+          anchor={anchor}
+          zIndex={zIndex}
+          className={openClass}
```

Comment:
Question: why target this using a `className`? You can use `${Content} {`

Reply:
No reasoning, guilty of being away from pre/post-processed CSS too much time that forgot to use many features it has 👍 
changed on https://github.com/artsy/palette/pull/1295/commits/842e521adcd786313b04c02df8c367c06fe699e5 (I also had to change the order of declarations to comply with linter rules)